packages/cozy-konnector-libs/src/libs/saveBills.js

```diff
@@ -32,53 +54,65 @@
  *
  * @module saveBills
  */
-
-const saveFiles = require('./saveFiles')
-const hydrateAndFilter = require('./hydrateAndFilter')
-const addData = require('./addData')
-const log = require('cozy-logger').namespace('saveBills')
-const linkBankOperations = require('./linkBankOperations')
-const DOCTYPE = 'io.cozy.bills'
-const _ = require('lodash')
-
-const requiredAttributes = {
-  date: 'isDate',
-  amount: 'isNumber',
-  vendor: 'isString'
-}
-
-// Encapsulate the saving of Bills : saves the files, saves the new data, and associate the files
-// to an existing bank operation
-module.exports = (entries, fields, options = {}) => {
+module.exports = async (entries, fields, options = {}) => {
   if (!_.isArray(entries) || entries.length === 0) {
     log('warn', 'saveBills: no bills to save')
     return Promise.resolve()
   }
 
-  checkRequiredAttributes(entries)
```

Comment:
We do not check Attributes anymore ?(as report in personne should not be remove)

Reply:
This is just a move